### ISSUE TITLE
CentOS 7 failed installation due to dependencies of phpMyAdmin-5.1.0~rc2-1.el7.remi.noarch

### ISSUE THREAD

**Issue by sickcodes** [serghey-rodin/vesta#2046]
### Operating System (OS/VERSION):

CentOS 7

### VestaCP Version:

http://vestacp.com/pub/vst-install.sh

### Installed Software (what you got with the installer):
```console
--nginx yes
--apache yes
--phpfpm no
--named no
--remi yes
--vsftpd no
--proftpd no 
--iptables yes 
--fail2ban yes 
--quota no 
--exim yes 
--dovecot yes 
--spamassassin no
--clamav yes 
--softaculous no
--mysql yes
--postgresql no
```
### Steps to Reproduce:

Type here, e.g. install vesta and type rm -rf / --no-preserve-root

### Related Issues/Forum Threads:
```console
--> Finished Dependency Resolution
Error: Package: phpMyAdmin-5.1.0~rc2-1.el7.remi.noarch (remi-test)
           Requires: php-symfony4-expression-language >= 4.4.9
           Available: php-symfony4-expression-language-4.1.13-1.el7.remi.noarch (remi)
               php-symfony4-expression-language = 4.1.13-1.el7.remi
           Installing: php-symfony4-expression-language-4.1.13-2.el7.remi.noarch (remi)
               php-symfony4-expression-language = 4.1.13-2.el7.remi
Error: Package: phpMyAdmin-5.1.0~rc2-1.el7.remi.noarch (remi-test)
           Requires: php-symfony4-config >= 4.4.9
           Available: php-symfony4-config-4.1.13-1.el7.remi.noarch (remi)
               php-symfony4-config = 4.1.13-1.el7.remi
           Available: php-symfony4-config-4.1.13-2.el7.remi.noarch (remi)
               php-symfony4-config = 4.1.13-2.el7.remi
Error: Package: phpMyAdmin-5.1.0~rc2-1.el7.remi.noarch (remi-test)
           Requires: php-symfony4-dependency-injection >= 4.4.9
           Available: php-symfony4-dependency-injection-4.1.13-1.el7.remi.noarch (remi)
               php-symfony4-dependency-injection = 4.1.13-1.el7.remi
           Available: php-symfony4-dependency-injection-4.1.13-2.el7.remi.noarch (remi)
               php-symfony4-dependency-injection = 4.1.13-2.el7.remi
Error: Package: phpMyAdmin-5.1.0~rc2-1.el7.remi.noarch (remi-test)
           Requires: php-symfony4-expression-language >= 4.4.9
           Available: php-symfony4-expression-language-4.1.13-1.el7.remi.noarch (remi)
               php-symfony4-expression-language = 4.1.13-1.el7.remi
           Available: php-symfony4-expression-language-4.1.13-2.el7.remi.noarch (remi)
               php-symfony4-expression-language = 4.1.13-2.el7.remi
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
Error: yum install failed
```

**Comment by JediMasterKenobi** [comment]
I'm having the same issue on Centos 7, I've been trying a couple differente solutions, one was to remove a few references to phomyadmin from the installer, it will give you errors but it will install vestacp, another issue is that imagick is not being installed, and it's quite essencial for software like wordpress.

**Comment by sickcodes** [comment]
I'm using HestiaCP on Ubuntu 20.04 now, as VestaCP appears to be abandoned.

**Comment by naurelijs** [comment]
Yup, same issue here

**Comment by rooteto** [comment]
we are experiencing the same problem.

**Comment by agbulutonur** [comment]
same here

**Comment by Mathster0598** [comment]
This is happening because of conflicts between the packages in the remi-test repo and the packages you have installed.

Once solution is to exclude the repo temporarily when updating using **yum --disablerepo=remi-test update**

or you could do **yum-config-manager --disable remi-test** to disable the repo permanently.
